### PR TITLE
[bitnami/jasperreports] Add VIB tests

### PR DESCRIPTION
**Pull request by marcosbc**
### Description of the change

The main objective of this PR is to publish our Bitnami JasperReports container using VMware Image Builder. In order to do that, several changes are included:

* Increasing the existing test coverage of the asset by adding Goss tests.
* Update verify and publish VIB pipeline's definitions.

### Benefits

* Ensuring higher quality of the container catalog.
* Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Applicable issues

N/A

### CODE REVIEW

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR @marcosbc! Please check my review comments whenever you have a chance.

File: .vib/jasperreports/goss/vars.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+binaries:
+  - java
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/jasperreports/goss/vars.yaml:2]: Good catch, added them.

File: .vib/jasperreports/goss/jasperreports.yaml
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/jasperreports/goss/jasperreports.yaml:1]: Added as well.

File: .vib/jasperreports/goss/vars.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      - /bitnami/jasperreports
+      - /opt/bitnami/jasperreports
+      - /opt/bitnami/jasperreports/apache-ant/bin
+      - /opt/bitnami/jasperreports/buildomatic
+      - /opt/bitnami/jasperreports/buildomatic/conf_source/db/mysql/jdbc
+      - /opt/bitnami/jasperreports/buildomatic/conf_source/db/postgresql/jdbc
+      - /opt/bitnami/jasperreports/buildomatic/install_resources/sql/mysql
+      - /opt/bitnami/jasperreports/buildomatic/install_resources/sql/postgresql
+files:
+  - paths:
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/jasperreports/goss/vars.yaml:15]: You are right, added it.

File: .vib/jasperreports/goss/vars.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  - mode: "0775"
+    paths:
+      - /bitnami/jasperreports
+      - /opt/bitnami/jasperreports
+      - /opt/bitnami/jasperreports/apache-ant/bin
+      - /opt/bitnami/jasperreports/buildomatic
+      - /opt/bitnami/jasperreports/buildomatic/conf_source/db/mysql/jdbc
+      - /opt/bitnami/jasperreports/buildomatic/conf_source/db/postgresql/jdbc
+      - /opt/bitnami/jasperreports/buildomatic/install_resources/sql/mysql
+      - /opt/bitnami/jasperreports/buildomatic/install_resources/sql/postgresql
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/jasperreports/goss/vars.yaml:13]: Those files are later used by the initialization logic. While we are not modifying them, if they are not present at those precise paths, the container will never be able to initialize properly.

File: .vib/jasperreports/goss/vars.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+files:
+  - paths:
+      - /opt/bitnami/jasperreports/buildomatic/conf_source/db/mysql/db.template.properties
+      - /opt/bitnami/jasperreports/buildomatic/conf_source/db/postgresql/db.template.properties
+      - /opt/bitnami/jasperreports/buildomatic/default_master.properties
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/jasperreports/goss/vars.yaml:18]: Sure, I'll add it.

File: .vib/jasperreports/goss/vars.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      - /opt/bitnami/jasperreports/buildomatic/js-ant
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/jasperreports/goss/vars.yaml:19]: You're right, added them.

[the remark this answers is not in the document]
Comment (joancafom) [reply .vib/jasperreports/goss/vars.yaml:26]: You added the files but forgot to check the permissions

**Review by joancafom — COMMENTED** [review]
Thanks for your PR! I left some minor comments 🚀

File: .vib/jasperreports/goss/vars.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (joancafom) [reply .vib/jasperreports/goss/vars.yaml:16]: Just checking it before proceeding to the approval 😁

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for your changes 🚀

**Pull request merged**